src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/AesSubtleCryptoTransform.Browser.cs

```diff
@@ -148,7 +148,7 @@ private unsafe int EncryptDecrypt(ReadOnlySpan<byte> input, Span<byte> output)
                     pOutput, output.Length);
 
                 if (bytesWritten < 0)
-                    throw new Exception(SR.Unknown_Error);
+                    throw new Exception(SR.Format(SR.Unknown_SubtleCrypto_Error, bytesWritten));
```

Comment:
I just noticed that these 3 are just `Exception`. @bartonjs - should they be `CryptographicException`?

Reply:
Oh, yep.  That's what we use when reporting "we called a native function for crypto and it failed", and, thus, by extension, almost every throw site (even if we've replaced something with managed code)